### ISSUE TITLE
Redundant documentation generated for empty constructor

### ISSUE THREAD

**Issue by bobhy** [godwinjk/KDoc-er#8]
I see automatically generated class/interface doc headers with a redundant line  "* create empty _name_", even if I didn't code a constructor with no parameters.  I know the compiler generates one even if I didn't code it, but it is usually just noise and I delete that line.

It was so annoying I coded up an option for the plugin to suppress the line  and have submitted a PR to enable it.

**Comment by CJCrafter** [comment]
I'll submit a new PR after #11 is merged for this, because I have the same issue and would love to see it resolved.
